charts/enterprise-metrics/values.yaml

```
@@ -1095,3 +1093,195 @@ minio:
       cpu: 100m
       memory: 128Mi
   secretKey: supersecret
+
+graphite:
+  # If true, enables graphite querier and graphite write proxy functionality.
+  # Read more in https://grafana.com/docs/metrics-enterprise/latest/graphite/
+  enabled: true
```

use a docstring which some tools recognize to generate helm docs


```suggestion
  # -- If true, enables graphite querier and graphite write proxy functionality.
  # Read more in https://grafana.com/docs/metrics-enterprise/latest/graphite/
  enabled: true
```

---

same applies to other fields below